amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -116,7 +116,7 @@ input GetUserTokenBalanceInput {
 input GetMotionStateInput {
   motionId: Int!
   colonyAddress: String!
-  transactionHash: String!
+  transactionHash: String
```

Comment:
Fetching motion state from an ActionListItem does not need to update staker rewards, so making this optional